src/app/checkout/step-3/step-3.tpl.html

```diff
@@ -202,9 +202,14 @@
                 <display-rate-totals rate-totals="$ctrl.cartData.frequencyTotals"></display-rate-totals>
               </div>
               <div class="checkout-cta pull-right">
-                <button id="submitOrderButton" type="submit" class="btn btn-primary btn-lg btn-block" ng-click="$ctrl.submitOrder()" ng-disabled="!$ctrl.canSubmitOrder()" translate>
-                  {{'SUBMIT_GIFT'}}
-                </button>
+                <recaptcha-wrapper action="'submit'"
+                           on-success="$ctrl.submitOrderInternal"
+                           component-instance="$ctrl.selfReference"
```

Comment:
It is passed because in the Angular controller, when called from the React component, has `this` being undefined, which is relied upon heavily by the success handler.